packages/react-storage/src/components/StorageManager/utils/getInput.ts

```diff
@@ -63,6 +58,11 @@ export const getInput = ({
       hasCallbackPath ? path({ identityId }) : path
     }${processedKey}`;
 
+    if (processFile) {
```

Comment:
Do we also need to add a call to `onProcessFileSuccess` inside the conditional block on [line 47](https://github.com/aws-amplify/amplify-ui/pull/5477/files#diff-04342a0ffee2005ce1b867d03610c88d0b63ec3b73a9c475fef0f66ef92d2f4aR47) for `key` inputs?

Reply:
Yep you're totally right